Add missing remap to make sure IMU and DVL data is extracted properly

icyflame opened the pull request:
- Data from the simulator is in the messages [sensor_msgs/Imu]() and [underwater_sensor_msgs/DVL]()
- Me and @aniket11bh have converted this message into the `kraken_msgs/imuData` and `kraken_msgs/dvlData` message formats.
- And then published these formats on the IMU and DVL topics.

FIXES #74 
FIXES #75 

TODO:
- Figure out why the `linear_acceleration` and `depth` values from the simulator IMU and DVL are not as expected
- Improve `pose_server_python`. See #77.

We have already tested this code, and the `pose_server_python` is working as we want it too. So, I will be merging this within an hour or two. I am opening this PR for posterity.

cc @prudhvid @kalyan-kumar @kshitijgoel007

icyflame committed:
Remove IMU remap, instead publish from remap.py

- The message type is converted from sensor_msgs/Imu to
  kraken_msgs/imuData.

Signed-off-by: Siddharth Kannan <kannan.siddharth12@gmail.com>

icyflame committed:
Add the remap code for linear acceleration.

Signed-off-by: Siddharth Kannan <kannan.siddharth12@gmail.com>

icyflame committed:
Add code to convert DVL message to kraken_msgs.

Signed-off-by: Siddharth Kannan <kannan.siddharth12@gmail.com>

icyflame committed:
Remap velocity variables per kraken_msgs dvlData

Signed-off-by: Siddharth Kannan <kannan.siddharth12@gmail.com>

prudhvid commented:
I guess your additions are there in https://github.com/auviitkgp/kraken_3.0/blob/indigo-devel/simulator_stack/simulator_to_kmsgs/src/main.cpp

icyflame commented:
@prudhvid Yeah, I think they are the same. But we have used an external package for the Quaternion to Euler conversion, and we have integrated it into `remap.py` itself, so, there was no requirement to change the `remap.launch` file. So, I think we can have this in the repo as well.

prudhvid commented:
Okay.

On Fri, Oct 2, 2015 at 3:13 AM, Siddharth Kannan <notifications@github.com>
wrote:

> @prudhvid <https://github.com/prudhvid> Yeah, I think they are the same.
> But we have used an external package for the Quaternion to Euler
> conversion, and we have integrated it into remap.py itself, so, there was
> no requirement to change the remap.launch file. So, I think we can have
> this in the repo as well.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/auviitkgp/kraken_3.0/pull/76#issuecomment-144858254>.
>



-- 
*Prudhvi Dharmana*

*B.Tech 4th year*
*Dept. of Computer Science & Engineering*
*IIT Kharagpur.*

icyflame committed:
Fix remap to remove thruster_force_converter

- Now, thruster_force_converter is not required anymore.
- The data published by control.py Node is directly subscribed to, by
  the remap node and then, remap publishes that to g500 's thruster
  input topic.

Signed-off-by: Siddharth Kannan <kannan.siddharth12@gmail.com>
